EDDI/ChangeLog.md

```diff
@@ -7,7 +7,7 @@
     * Hardened EDDI against a crash that could occur if the folder containing player journals doesn't exist.
     * Smarter vehicle state tracking.
   * Speech Responder
-    * Add new event 'Vehicle destroyed' *(it does not perfectly track vehicle destruction since there are no official player journal events for SRV or fighter destruction - we have to infer vehicle destruction)*.
+    * Add new event 'Vehicle destroyed' *(it does not perfectly track vehicle destruction since there are no official player journal events for SRV or fighter destruction - we have to infer vehicle destruction. Still, it's better than nothing.)*.
```

Comment:
I'll let it slide, and I've done it myself a few times, but in general it's a bad idea to include unrelated changes in a feature branch.